### PR TITLE
Fix pytype

### PR DESCRIPTION
**Pull request by pawloch00**
## Fixes / Features
-
-

## Testing / Documentation
Testing details.

- [ y/n ] Tests pass
- [ y/n ] Appropriate changes to documentation are included in the PR

### CODE REVIEW

File: .github/workflows/lint_and_format.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ jobs:
         python -m pip install pyink==24.3.0
         pip install pytype
         pip install pylint
+        pip install .
```
Comment (mbobrovskyi) [inline .github/workflows/lint_and_format.yml:39]: Hmm. But why didn’t we catch this error in the original PR?

Comment (mbobrovskyi) [reply .github/workflows/lint_and_format.yml:39]: I also can see linter errors on main branch.

Comment (pawloch00) [reply .github/workflows/lint_and_format.yml:39]: Not sure, probably I tried it on file not importing urllib

Comment (pawloch00) [reply .github/workflows/lint_and_format.yml:39]: What kind of errors? Aim of this change and previous PR is to add linter checks for new code. For existing code type hints will be corrected in separate PR

Comment (mbobrovskyi) [reply .github/workflows/lint_and_format.yml:39]: I mean after push changes github action didn't show us this error.

Comment (mbobrovskyi) [reply .github/workflows/lint_and_format.yml:39]: Probably we need to revert `push`. https://github.com/AI-Hypercomputer/xpk/pull/227/files#diff-9be398388c4a96355ba1dcbf7c41cee8567621b5e50c1693da04a5639394ce42L17

Comment (pawloch00) [reply .github/workflows/lint_and_format.yml:39]: yep, aim of this pr is to remove error comming from missing urllib dependency

Comment (pawloch00) [reply .github/workflows/lint_and_format.yml:39]: I see, probably we need to fix it, before we have each check run twice(first time with pull request trigger and second time with push trigger)

Comment (mbobrovskyi) [reply .github/workflows/lint_and_format.yml:39]: Also, we need to check the trigger for merges to the main branch.

**Review by mbobrovskyi — APPROVED** [review]
LGTM. Thanks for fix!

**Pull request merged**